src/Microsoft.Net.Http.Headers/EntityTagHeaderValue.cs

```diff
@@ -112,6 +112,24 @@ public override int GetHashCode()
             return _tag.GetHashCode() ^ _isWeak.GetHashCode();
         }
 
+        public static bool CompareStrongly(EntityTagHeaderValue left, EntityTagHeaderValue right)
+        {
+            if (left == null || right == null)
+            {
+                return false;
+            }
+            return !left.IsWeak && !right.IsWeak && string.Equals(left.Tag, right.Tag);
+        }
+
+        public static bool CompareWeakly(EntityTagHeaderValue left, EntityTagHeaderValue right)
+        {
+            if (left == null || right == null)
+            {
+                return false;
+            }
+            return string.Equals(left.Tag, right.Tag);
```

Comment:
Guess I'll go update all the instances in ResponseCaching where I relied on the default value.